google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -279,10 +279,10 @@ def _get_query_argument(args, cell, env):
 
 
 def _sample_cell(args, cell_body):
-  """Implements the bigquery sample cell magic for ipython notebooks.
+  """Implements the bq sample cell magic for ipython notebooks.
```

Comment:
Maybe it should be "Implements the BigQuery sample" so it is consistent with others?